### ISSUE TITLE
Mail Rule for Forwarding

### ISSUE THREAD

**Issue by ginfuru** [nylas/N1#2229]
`FEATURE REQUEST`
It would be super helpful if I could set up a mail rule to forward to another address. 

my use case example would be a help desk and support ticket systems. 

email comes to `support@yourdomain.com` and then is forwarded to `yourCompanyFreshDesk@yourCompany.somehelpdesk.com` generates a support ticket inside of the systems.

**Comment by brunohq** [comment]
Yes, I'd love to have this feature as well! Especially when working with a team or a VA.
